app/src/components/new-request/stats.vue

```diff
@@ -111,11 +111,22 @@ export default class Stats extends Vue {
 
   async created (): Promise<void> {
     if (
+    /*
+     * Feature-flag semantics for hardcoded_*_wait_time:
+     * - flag > 0 : use the flag value (explicit positive wait time).
+     * - flag === 0 OR FF disabled: fall back to backend stats.
```

Comment:
If the FF is "disabled", it's still providing a value.

I think it makes sense for the "disabled" value to be 0. It's like that right now:
<img width="762" height="143" alt="image" src="https://github.com/user-attachments/assets/6de8a490-e6a3-485d-8b48-d215581be886" />